src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs

```diff
@@ -60,7 +61,7 @@ public void OneTimeSetUp()
                 httpClient,
                 new PaymentOrdersResource(httpClient),
                 new ConsumersResource(httpClient),
-                new PaymentsResource(httpClient, new CardPaymentsResource(httpClient), new SwishPaymentsResource(httpClient)));
+                new PaymentsResource(httpClient, new CardPaymentsResource(httpClient), new SwishPaymentsResource(httpClient), new MobilePayPaymentsResource(httpClient)));
```

Comment:
If the default implementations are used, `PaymentsResource` should be able to construct them itself.

```suggestion
                new PaymentsResource(httpClient));
```